recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-devtools/clang/compiler-rt_git.bbappend

```diff
@@ -0,0 +1 @@
+TUNE_CCARGS := "${@oe.utils.str_filter_out('-mtune=cortex-a57.cortex-a53', '${TUNE_CCARGS}', d)}"
```

Comment:
In general, you can try to rewrite build rules and provide  CROSS_COMPILE=  flag to clang, which should point to a gcc (linaro arm gcc)compiler, which can handle such specific flags...